workshops/website_mockup_generator/README.md

```diff
@@ -0,0 +1,485 @@
+
+You can optionally print the URL of the mockup image using :
+
+```javascript
+console.log(mockup);
+```
+
+Now, save the path to the mockup in `resultImg.src`:
+```
+  resultImg.src = mockup;
+```
+
+Your code should be similar to this as of now :
+
+```javascript
+const resultImg = document.getElementById("resultImg");
+const Download = document.getElementById("Download");
+
+const generateMockup = () => {
+  const websiteAdress = document.getElementById("websiteAdress").value;
+  const bgColor = document.getElementById("bgColor").value;
+  const mockup ="https://2s9e3bif52.execute-api.eu-central-1.amazonaws.com/production/screenshot?url=" +websiteAdress +"&color=" +bgColor;
+  console.log(mockup);
+  resultImg.src = mockup;
+};
+```
+
+#### Adding the download functionality
+To allow users to download the mockup image by pressing the download button, include the following code inside the `generateMockup()` function.
+
+```javascript
+Download.addEventListener("click", function () {
+window.open(mockup);
+});
+```
+
+The above code will open the mockup image for the user to download it.
+
+<details><summary>Your final  JavaScript code should look something like this:</summary>
+  
+```javascript
+const resultImg = document.getElementById("resultImg");
+const Download = document.getElementById("Download");
+
+
+const generateMockup = () => {
+  const websiteAddress = document.getElementById("websiteAddress").value;
+  const mockup =
+    "https://2s9e3bif52.execute-api.eu-central-1.amazonaws.com/production/screenshot?url=" +
+    websiteAdress;
+  console.log(mockup);
+  resultImg.src = mockup;
+
+  Download.addEventListener("click", function () {
+    window.open(mockup);
+  });
+};
+```
+</details>
+
+And finally, the coding part is done!
+
+
+
+## Part 5: The End
+
+You can try running the code by pressing the *'Run'* button on the top. You will see the preview of the website mockup generator in the right side pane.
+
+![Pressing the run button](https://cloud-kk7f3ujxb.vercel.app/0mockuprun.gif)
+
+Hurray! Cheers on building your own website mockup generator.
+
+![Man celebrating](https://media.giphy.com/media/6nuiJjOOQBBn2/giphy.gif)
+
+Here are some of the mockups that I made :
+
+![Mockup of my portfolio](https://cloud-39p70jl8k.vercel.app/0mockup2.png)
+
+![Mockup of Hack Club website](https://cloud-etbe898rq.vercel.app/0mockup1.png)
+
+
+## What's next? 🚀
+
+![Thinking man](https://media.giphy.com/media/d3mlE7uhX8KFgEmY/giphy.gif)
+
+You are free to customize and contribute to the website mockup generator! You can consider trying out the below if you are interested :
+
+- ![A retro style mockup generator](https://repl.it/@19EUCS071KUMARA/Website-Mockup-Generator)
+- ![A minimalistic mockup generator](https://repl.it/join/dlpvaodr-19eucs071kumara)
+- ![A dark themed mockup generator](https://repl.it/@19EUCS071KUMARA/Website-Mockup-Generator-1)

```

Comment:
no idea why, but images aren't rendering

Reply:
yeah fixed them..they're actually links not pics